cmd/workflows.go

```diff
@@ -219,50 +216,85 @@ func generateServiceAccounts(namespace *corev1.Namespace, roleBindingLister rbac
 		return nil, err
 	}
 
-	// Service account for UI access
+	// Setup the default service account
 	serviceAccounts = append(serviceAccounts, &corev1.ServiceAccount{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        fmt.Sprintf("argo-workflows-%v", namespace.Name),
-			Namespace:   workflowsSystemNamespace,
-			Annotations: map[string]string{"workflows.argoproj.io/rbac-rule": fmt.Sprintf("'%s' in groups", roleBinding.Subjects[0].Name)},
+			Name:      "user-default-login",
```

Comment:
Is the default one supposed to be per-namespace or globally? I thought when reading the docs it was one globally but I could be wrong.